### PR TITLE
how-to: update ansible deployment doc

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? 

Update and refine the language of the Ansible deployment document.

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs-cn/pull/2117

### Which version does your change affect? 

dev, v2.1, v3.0, v3.1

### CODE REVIEW

**Comment by lilin90** [comment]
@TomShawn I made more updates in https://github.com/pingcap/docs-cn/pull/2117. Please check it and update it accordingly where necessary. Thanks!

**Comment by TomShawn** [comment]
> @TomShawn I made more updates in [pingcap/docs-cn#2117](https://github.com/pingcap/docs-cn/pull/2117). Please check it and update it accordingly where necessary. Thanks!

OK.

**Comment by lilin90** [comment]
@TomShawn Please resolve conflicts.

File: dev/how-to/deploy/orchestrated/ansible.md [hunk 1]
```diff
@@ -180,7 +180,7 @@ It is required to use `pip` to install Ansible and its dependencies, otherwise a
     sudo pip install -r ./requirements.txt
     ```
 
-    Ansible and the related dependencies are in the `tidb-ansible/requirements.txt` file.
+    Ansible and the version information dependencies are in the `tidb-ansible/requirements.txt` file.
```
Comment (lilin90) [inline dev/how-to/deploy/orchestrated/ansible.md]:
```suggestion
    The version information of Ansible and dependencies is in the `tidb-ansible/requirements.txt` file.
```

File: dev/how-to/deploy/orchestrated/ansible.md [hunk 2]
```diff
@@ -333,11 +333,11 @@ You can use either of the following two methods to change the governor mode:
 
 Log in to the target machines using the `root` user account.
 
-Format your data disks to the ext4 filesystem and mount the filesystem with the `nodelalloc` and `noatime` options. It is required to mount the `nodelalloc` option, or else the Ansible deployment cannot pass the test. The `noatime` option is optional.
+Format your data disks to the ext4 filesystem and add the `nodelalloc` and `noatime` mount options to the filesystem. It is required to add the `nodelalloc` option, or else the Ansible deployment cannot pass the test. The `noatime` option is optional.
```
Comment (lilin90) [inline dev/how-to/deploy/orchestrated/ansible.md]:
```suggestion
Format your data disks to the ext4 filesystem and add the `nodelalloc` and `noatime` mount options to the filesystem. It is required to add the `nodelalloc` option, or else the Ansible deployment cannot pass the test. The `noatime` option is optional.
```

**Comment by TomShawn** [comment]
@lilin90 Comments addressed, PTAL again, thanks!

**Pull request merged**